libs/application/templates/health-insurance/src/lib/HealthInsuranceTemplate.ts

```diff
@@ -26,11 +26,15 @@ const HealthInsuranceSchema = z.object({
     nationality: z.string().nonempty(),
     email: z.string().email(),
     phoneNumber: z.string().optional(),
+    country: z.string().nonempty(),
```

Comment:
I think we can lift these three outside of applicant. I am thinking of renaming the `applicant` to `contactInfo` instead so it doesn't add any confusions on adding new fields as everything in this field is about the applicant.